Frontend/ui-customer/src/Flow.purs

```diff
@@ -1097,6 +1097,22 @@ homeScreenFlow = do
                                                   }) srcSpecialLocation.gates
             if (sourceServiceabilityResp.serviceable ) then do
               let cityName = getCityNameFromCode sourceServiceabilityResp.city
+              (GlobalState currentState) <- getState
+              PlaceName address <- getPlaceName currentState.homeScreen.props.sourceLat currentState.homeScreen.props.sourceLong HomeScreenData.dummyLocation false
+              if address.formattedAddress /= "NO_LOCATION_FOUND" then do
```

Comment:
For better type safety, can we make this Nothing if there no address found("NO_LOCATION_FOUND"), instead of returning a string.